<ui version="4.0" >
 <class>CoverOptionsPage</class>
 <widget class="QWidget" name="CoverOptionsPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>314</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="rename_files" >
     <property name="title" >
      <string>Location</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>2</number>
      </property>
      <item>
       <widget class="QCheckBox" name="save_images_to_tags" >
        <property name="text" >
         <string>Save cover images to tags</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="save_images_to_files" >
        <property name="text" >
         <string>Save cover images as separate files</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="cover_image_filename" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="rename_files_2" >
     <property name="title" >
      <string>Amazon</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>2</number>
      </property>
      <item>
       <widget class="QCheckBox" name="use_amazon_images" >
        <property name="text" >
         <string>Use cover images from Amazon</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Tahoma'; font-size:8pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-style:italic;">TODO: add warning about Amazon's ToS&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>275</width>
       <height>51</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>save_images_to_tags</tabstop>
  <tabstop>save_images_to_files</tabstop>
  <tabstop>cover_image_filename</tabstop>
  <tabstop>use_amazon_images</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
